modules/core/shared/src/main/scala/gem/Track.scala

```diff
@@ -51,11 +65,27 @@ object Track {
       ephemerides.get(s)
 
   }
+
+  @SuppressWarnings(Array("org.wartremover.warts.PublicInference"))
   object Nonsidereal {
 
     def empty(key: EphemerisKey): Nonsidereal =
       Nonsidereal(key, Map.empty)
 
+    implicit val EqNonsidereal: Eq[Nonsidereal] =
+      Eq.fromUniversalEquals
+
   }
 
+  val sidereal: Prism[Track, Sidereal] =
+    GenPrism[Track, Sidereal]
+
+  val nonsidereal: Prism[Track, Nonsidereal] =
+    GenPrism[Track, Nonsidereal]
+
+  val ephemerides: Optional[Track, Map[Site, Ephemeris]] =
+    nonsidereal composeLens Nonsidereal.ephemerides
+
+  implicit val EqTrack: Eq[Track] =
+    Eq.fromUniversalEquals
```

Comment:
What does this do?

Reply:
This constructs an `Eq` instance that delegates to universal equality (i.e, Scala `==`). This is ethical for case classes whose members also have sane universal equality.